bin/cli.js

```diff
@@ -18,16 +18,7 @@
 
 	const ErrorHelpers = require("./utils/errorHelpers");
 
-	const NON_COMPILATION_ARGS = [
-		"init",
-		"migrate",
-		"add",
-		"remove",
-		"serve",
-		"generate-loader",
-		"generate-plugin",
-		"info"
-	];
+	const { NON_COMPILATION_ARGS } = require("./constants");
```

Comment:
We should move it at the top of the file

Reply:
Done!